dateparser/custom_language_detection/language_mapping.py

```diff
@@ -0,0 +1,18 @@
+from dateparser.data.languages_info import language_map
+
+
+def map_languages(language_codes):
+    """
+    Returns the candidates from the supported languages codes.
+    :param languages:
```

Comment:
This gives the following warning:
`Function 'map_languages' does not have a parameter 'languages'`
We need to update the docstring comment.

Reply:
Done 👍🏻